rosidl_typesupport_c/resource/srv__type_support.cpp.em

```diff
@@ -17,8 +17,8 @@ TEMPLATE(
 
 @{
 from rosidl_cmake import convert_camel_case_to_lower_case_underscore
-include_parts = [package_name] + list(interface_path.parents[0].parts) + \
-    [convert_camel_case_to_lower_case_underscore(interface_path.stem)]
+include_parts = [package_name] + list(interface_path.parents[0].parts) + [
+    'detail', convert_camel_case_to_lower_case_underscore(interface_path.stem)]
```

Comment:
It doesn't look like `include_parts` is only used at L22 to populate `include_base`, which itself is not used until it is overridden at L124. I guess this logic can safely be removed.

Reply:
Done in 823621a971d357951f2af0ba4154f664040f56fc.